src/Adapter/SymfonyContainer.php

```diff
@@ -47,8 +48,12 @@ public static function getInstance()
     {
         if (!isset(self::$instance)) {
             global $kernel;
-
-            if (!is_null($kernel) && $kernel instanceof KernelInterface) {
+            if ($kernel === null) {
+                $kernel = new AppKernel(_PS_MODE_DEV_ ? 'dev' : 'prod', _PS_MODE_DEV_);
```

Comment:
is there any constant to define 'dev' and 'prod' ?

Reply:
of course not :trollface: